components/coding/GoalsSectionComponent.tsx

```diff
@@ -1,18 +1,32 @@
-import { PencilAltIcon, PencilIcon } from "@heroicons/react/outline";
+import { PencilAltIcon } from "@heroicons/react/outline";
 import { differenceInCalendarDays, format } from "date-fns";
-import moment from "moment";
 import Link from "next/link";
-import React from "react";
+import React, { useState } from "react";
 import { UserGoalsData } from "../../graphql/fetchUserGoals";
-
 export type GoalsSectionProps = {
-  userGoals: UserGoalsData[];
-  sectionName: string;
+  sectionName?: string;
+  userGoals?: UserGoalsData[];
+};
+
+export const returnGoalStyle = (goal) => {
```

Comment:
let's define the type for the function input so Typescript knows what's available on goal.. 

i.e const returnGoalStyle = (goal: UserGoalsData)...

Reply:
thank you! did that.